src/Compilers/Core/Portable/PEWriter/SigningUtilities.cs

```diff
@@ -27,11 +27,15 @@ internal static byte[] CalculateRsaSignature(IEnumerable<Blob> content, RSAParam
             }
         }
 
-        private static byte[] CalculateSha1(IEnumerable<Blob> content)
+        internal static byte[] CalculateSha1(IEnumerable<Blob> content)
         {
             using (var hash = IncrementalHash.CreateHash(HashAlgorithmName.SHA1))
             {
-                hash.AppendData(content);
```

Comment:
I didn't understand this change. 
Was there an overload of `AppendData` that takes an `IEnumerable`? I can't find it on MSDN.
If yes, what was wrong with that method?